apis/python/tests/test_basic_anndata_io.py

```diff
@@ -123,11 +123,11 @@ def test_import_anndata(adata, ingest_modes):
             with pytest.raises(ValueError):
                 exp.ms["RNA"].X["data"].read(coords=all2d)
 
-        # Check raw/X/data (sparse)
+        #        # Check raw/X/data (sparse)
         assert exp.ms["raw"].X["data"].metadata.get(metakey) == "SOMASparseNDArray"
         if have_ingested:
-            matrix = exp.ms["raw"].X["data"].read(coords=all2d).coos().concat()
```

Comment:
please leave all support and tests for COO tensors in place.  They should _not_ be removed.

Adding tests for `tables` alongside COO Tensor is great!

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/952#discussion_r1108907410

Reply:
Restored on most recent commit as discussed above